lms/templates/courseware/courseware-chromeless.html

```diff
@@ -59,37 +62,16 @@
   % endif
 
   <script type="text/javascript">
-    var $$course_id = "${course.id | escapejs}";
-
-    $(function(){
-        $(".ui-accordion-header a, .ui-accordion-content .subtitle").each(function() {
```

Comment:
@andy-armstrong: FYI: Made all matching changes in courseware-chromeless.html, including this one.  Thanks for alerting me to the nice code duplication. :)